src/DynamoCore/DynamoCore.csproj

```diff
@@ -33,7 +33,8 @@
     <PackageReference Include="Autodesk.IDSDK" Version="1.2.4" />
     <PackageReference Include="Greg" Version="3.0.2.6533" />
     <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.6705" GeneratePathProperty="true" />
```

Comment:
If this is meant for debugging libg on Linux, we may also need a debug package for 230, at least until we update libg to 231 on DaaS.

Reply:
Ugh, if possible ai would prefer to just switch DAAS to 231. If no breaking changes are expected